Ambient test for `test_change_custom_images` is flaky

### Bug Description

In many CI runs, the test `test_change_custom_images` (KFP Profile Controller) fails with the logs provided below.
Usually a re-trigger is enough to get it working.

Example failing run can be found [here](https://github.com/canonical/kfp-operators/actions/runs/27675445981/job/81850358079).

### To Reproduce

Run CI action multiple times

### Environment

CI

### Relevant Log Output

```shell
FAILED tests/integration/test_charm_ambient.py::test_change_custom_images - AssertionError: assert 'docker.io/charmedkubeflow/frontend:2.16.0-c9e8334' == 'gcr.io/ml-pipeline/frontend:latest'
```

### Additional Context

_No response_

--- a/charms/kfp-profile-controller/tests/integration/test_charm_ambient.py
+++ b/charms/kfp-profile-controller/tests/integration/test_charm_ambient.py
@@ -326,9 +326,11 @@ def validate_profile_resources(
     assert expected_label_value == namespace.metadata.labels[expected_label]
 
 
+# Retry for 90 seconds since the metacontroller's `resyncPeriodSeconds` is
+# currently 60s, so at worst it will take 60s for the resync to be triggered.
 @retry(
     wait=wait_exponential(multiplier=1, min=1, max=10),
-    stop=stop_after_delay(60),
+    stop=stop_after_delay(90),
     reraise=True,
 )
 def validate_profile_deployments_with_custom_images(

--- a/charms/kfp-profile-controller/tests/integration/test_charm_object_storage.py
+++ b/charms/kfp-profile-controller/tests/integration/test_charm_object_storage.py
@@ -232,9 +232,11 @@ def validate_profile_resources(
     assert expected_label_value == namespace.metadata.labels[expected_label]
 
 
+# Retry for 90 seconds since the metacontroller's `resyncPeriodSeconds` is
+# currently 60s, so at worst it will take 60s for the resync to be triggered.
 @retry(
     wait=wait_exponential(multiplier=1, min=1, max=10),
-    stop=stop_after_delay(60),
+    stop=stop_after_delay(90),
     reraise=True,
 )
 def validate_profile_deployments_with_custom_images(

--- a/charms/kfp-profile-controller/tests/integration/test_charm_s3.py
+++ b/charms/kfp-profile-controller/tests/integration/test_charm_s3.py
@@ -321,9 +321,11 @@ def validate_profile_resources(
     assert expected_label_value == namespace.metadata.labels[expected_label]
 
 
+# Retry for 90 seconds since the metacontroller's `resyncPeriodSeconds` is
+# currently 60s, so at worst it will take 60s for the resync to be triggered.
 @retry(
     wait=wait_exponential(multiplier=1, min=1, max=10),
-    stop=stop_after_delay(60),
+    stop=stop_after_delay(90),
     reraise=True,
 )
 def validate_profile_deployments_with_custom_images(

